src/stores/ui.ts

```diff
@@ -139,5 +141,16 @@ export const useUiStore = defineStore('ui', {
     toggleFilters() {
       this.setFiltersState(!this.isFilterVisible)
     },
+    dismissBanner(bannerId: string) {
+      this.dismissedBanners.push(bannerId)
+      this.$nuxt.$cookies.set(
+        'uiDismissedBanners',
+        this.dismissedBanners,
+        cookieOptions
+      )
+    },
+    isBannerDismissed(bannerId: string): boolean {
+      return this.dismissedBanners.includes(bannerId)
+    },
```

Comment:
I suppose these could use the `BannerId` type as well?

```suggestion
    dismissBanner(bannerId: BannerId) {
      this.dismissedBanners.push(bannerId)
      this.$nuxt.$cookies.set(
        'uiDismissedBanners',
        this.dismissedBanners,
        cookieOptions
      )
    },
    isBannerDismissed(bannerId: BannerId): boolean {
      return this.dismissedBanners.includes(bannerId)
    },
```